region/norfair/east/Norfair Reserve Tank Room.json

```diff
@@ -151,6 +151,53 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "Collect Both Items with Pause Abuse",
+      "entranceCondition": {
+        "comeInRunning": {
+          "minTiles": 2,
+          "speedBooster": "any"
+        }
+      },
+      "requires": [
+        {"or": [
+          "Wave",
+          "Plasma",
+          "Spazer",
+          "canBeVeryPatient"
+        ]},
+        {"enemyKill": {
+          "enemies": [
+            ["Sova", "Sova", "Sova", "Dragon", "Dragon"]
+          ],
+          "explicitWeapons": ["PowerBomb"]
+        }},
+        "canMockball",
+        "canPreciseWalljump",
+        "canInsaneJump",
+        "canPauseAbuse",
+        "canFarmWhileShooting",
+        "canBePatient",
+        {"resourceMissingAtMost": [
+          {"type": "Missile", "count": 0},
+          {"type": "Super", "count": 0}]},
+        {"heatFrames": 392},
+        {"resourceAtMost": [{"type": "Energy", "count": 1}]}
+      ],
+      "collectsItems": [2, 3],
+      "note": [
+        "Very quickly cross the room, collecting both items without falling into the lava.",
+        "Collect Sova and Dragon drops on the way back to make use of pause abuse to save more energy.",
+        "A Power Bomb in the middle of the room sets up the Sova drops and adds lag which makes the block item easier to collect with a Flatley jump.",
+        "Farming the left sova and middle dragon increases the chances of getting enough health drops, but it may still not be possible with poor drop luck."
```

Comment:
```suggestion
        "A Power Bomb in the middle of the room sets up the Sova drops and adds lag which makes the block item easier to collect with a Flatley jump.",
        "Farming the left sova and middle dragon increases the chances of getting enough health drops, but it may still not be possible with poor drop luck.",
        "Pause abuse can be useful for reaching the right side drops if the left side's luck was bad."
```